packages/as-transformer/src/helpers/node.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  FunctionDeclaration,
+  IdentifierExpression,
+} from 'assemblyscript/dist/assemblyscript.js';
+
+// inspired from https://github.com/as-pect/visitor-as/blob/master/src/utils.ts#L35
+// TODO: open an issue on visitor-as and link it here.
```

Comment:
What does not work with the hasDecorator function from the visitor-as lib ?

Reply:
I did similar thing for wit and I think you can get the decorator from node.decorators

Reply:
I have the following error:
```
FAILURE TypeError: Cannot read properties of undefined (reading 'decorators')
    at Module.hasDecorator (/home/greg/tmp/tmp2/as/node_modules/visitor-as/src/utils.ts:46:15)
    at Transformer.visitFunctionDeclaration (file:///home/greg/tmp/tmp2/as/packages/as-transformer/dist/transformer.js:13:19)
    at Transformer._visit (/home/greg/tmp/tmp2/as/node_modules/visitor-as/src/baseTransform.ts:257:21)
    at Transformer.visitSource (/home/greg/tmp/tmp2/as/node_modules/visitor-as/src/baseTransform.ts:310:28)
    at Transformer._visit (/home/greg/tmp/tmp2/as/node_modules/visitor-as/src/baseTransform.ts:90:21)
    at Transformer.visit (/home/greg/tmp/tmp2/as/node_modules/visitor-as/src/visitor.ts:62:19)
    at file:///home/greg/tmp/tmp2/as/packages/as-transformer/dist/transformer.js:33:18
    at Array.forEach (<anonymous>)
    at Transformer.afterParse (file:///home/greg/tmp/tmp2/as/packages/as-transformer/dist/transformer.js:31:17)
    at Oe (/home/greg/tmp/my-sc/node_modules/assemblyscript/cli/index.js:476:27)
    at Mn (/home/greg/tmp/my-sc/node_modules/assemblyscript/cli/index.js:698:17)
    at asp (/home/greg/tmp/my-sc/node_modules/@as-pect/cli/src/index.ts:173:22)

```